<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IsbTranslator</class>
 <widget class="QMainWindow" name="IsbTranslator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IsbTranslator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTextEdit" name="SourceEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="translateButton">
        <property name="text">
         <string>Translate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextEdit" name="TargetEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QComboBox" name="comboBoxSource"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxTarget"/>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>translateButton</zorder>
   <zorder>TargetEdit</zorder>
   <zorder>comboBoxTarget</zorder>
   <zorder></zorder>
   <zorder></zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
